src/views/projectExplorer/index.ts

```diff
@@ -54,6 +56,66 @@ export default class ProjectExplorer implements TreeDataProvider<ProjectExplorer
           }
         }
       }),
+      commands.registerCommand(`vscode-ibmi-projectexplorer.projectExplorer.migrateToIFS`, async (element: Library) => {
+        if (element) {
+          const migrationData = await migrateToIFS(element.label!.toString());
+          if (migrationData) {
+            const ibmi = getInstance();
+            let deploymentDirs = ibmi?.getStorage().getDeployment()!;
+            let remoteDir = deploymentDirs[element.workspaceFolder.uri.fsPath];
+            if (!remoteDir) {
+              await commands.executeCommand(`code-for-ibmi.setDeployLocation`, undefined, element.workspaceFolder, `${ibmi?.getConfig().homeDirectory}${element.workspaceFolder.name}`);
+
+              deploymentDirs = ibmi?.getStorage().getDeployment()!;
+              remoteDir = deploymentDirs[element.workspaceFolder.uri.fsPath];
+              if (!remoteDir) {
+                return;
+              }
+            }
+
+            let migration = {
+              numFiles: migrationData.sourceFiles.length,
+              numSuccess: 0,
+              numFail: 0,
+              result: true
+            };
+            await window.withProgress({
+              location: ProgressLocation.Notification,
+              title: l10n.t('Migrating to IFS'),
+            }, async (progress) => {
+              for await (const file of migrationData.sourceFiles) {
+                progress.report({ message: file });
+
+                const result = await ibmi?.getConnection().sendCommand({
+                  command: `export PATH="/QOpenSys/pkgs/bin:$PATH:" && /QOpenSys/pkgs/bin/makei cvtsrcpf -c ${migrationData.defaultCCSID} ${path.parse(file).name} ${element.label}`,
```

Comment:
We likely need to add a check before this runs to ensure `/QOpenSys/pkgs/bin/makei` is installed/exists.

Reply:
Good point, added this check